herd/AArch64ASLSem.ml

```diff
@@ -730,6 +730,8 @@ module Make (TopConf : AArch64Sig.Config) (V : Value.AArch64ASL) :
                   "domain" ^= var (barrier_domain dom);
                   "types" ^= var (barrier_typ btyp);
                 ] )
+      | I_UDF k when C.variant Variant.ASL_AArch64_UDF ->
```

Comment:
Out of curiosity, can't unconditionally handle UDF with that ASL code? Because if that's the case you don't really need the variant either, do you?

Reply:
ah yes, I think this something that happens in decode.